keras/src/quantizers/gptqutils.py

```diff
@@ -0,0 +1,419 @@
+import io
+import random
+import tarfile
+
+import numpy as np
+import requests
+from absl import logging
+from datasets import load_dataset
+
+from keras.src import ops
+from keras.src.layers import Dense
+from keras.src.layers import EinsumDense
+from keras.src.layers import Embedding
+from keras.src.quantizers.gptq import GPTQ
+from keras.src.quantizers.gptqquant import GPTQQuant
+
+
+def get_dataloader(tokenizer, seqlen, dataset, nsamples=128, seed=0):
```

Comment:
NIT seqlen->sequence_len

Reply:
I think seqlen is standard and is used everywhere